### PR TITLE
cleanup: define inline ns to be GOOGLE_CLOUD_CPP_NS

### PR DESCRIPTION
**Pull request by devjgm**
This is a cleanup related to #5976

There's no semantic change here.

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7204)

### CODE REVIEW

File: google/cloud/pubsub/version.h [hunk 1]
```diff
@@ -18,32 +18,6 @@
 #include "google/cloud/pubsub/version_info.h"
 #include "google/cloud/version.h"
 
-#define GOOGLE_CLOUD_CPP_PUBSUB_NS                                     \
-  GOOGLE_CLOUD_CPP_VEVAL(GOOGLE_CLOUD_CPP_PUBSUB_CLIENT_VERSION_MAJOR, \
-                         GOOGLE_CLOUD_CPP_PUBSUB_CLIENT_VERSION_MINOR)
-
-namespace google {
-/**
```
Comment (coryan) [inline google/cloud/pubsub/version.h:26]: I can see why you would remove these, but they show up in the documentation for Pub/Sub:

https://googleapis.dev/cpp/google-cloud-pubsub/latest/namespaces.html

If you remove them from here, the documentation will be gone.

**Review by coryan — COMMENTED** [review]
Where does Doxygen get the namespace documentation from now?

File: google/cloud/pubsub/version.h (hunk 1 above)
Comment (devjgm) [reply google/cloud/pubsub/version.h:26]: Oh, I didn't know that was used for anything. Added them back.

Comment (coryan) [reply google/cloud/pubsub/version.h:26]: It is possible they are redundant, but let's separate that cleanup if that is the case.

**Review by coryan — APPROVED** [review]

**Pull request merged**